csrc/codegen.cpp

```diff
@@ -1652,20 +1652,16 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
         .append("(*)[")
         .append(items_per_thread)
         .append("])")
-        .append("(")
-        .append(
-            genVariableName(output_values) + ".array + " +
-            genInline(output_values->index()))
+        .append("(&")
```

Comment:
Simplifying generated code a bit:

Before:
```
xyz + 0
```

After:
```
&xyz[0]
```

Reply:
happen to notice at line 1631, should it be `sorted_parallel_types.get(pt)` instead of `parallel_dimension_map.get(pt)`?